include/Graphs/PAG.h

```diff
@@ -153,6 +153,13 @@ class PAG : public GenericGraph<PAGNode,PAGEdge>
         }
         return pag;
     }
+
+    /// Return pointer to PAG if it has been built, otherwise nullptr.
+    static inline PAG* tryGetPAG()
+    {
+        return pag;
+    }
+
```

Comment:
I don't think we do need this method. You can make the option `IncludePAGInICFGDump` as false by default. It will solve your problem.